proof/crefine/AARCH64/Wellformed_C.thy

```diff
@@ -86,8 +86,10 @@ abbreviation
 abbreviation
   vs_Ptr :: "machine_word \<Rightarrow> vs_ptr" where "vs_Ptr == Ptr"
 
+value_type vgic_lr_len = "max_armKSGICVCPUNumListRegs"
+
 abbreviation
-  vgic_lr_C_Ptr :: "addr \<Rightarrow> (virq_C[64]) ptr" where "vgic_lr_C_Ptr \<equiv> Ptr"
+  vgic_lr_C_Ptr :: "addr \<Rightarrow> (virq_C[vgic_lr_len]) ptr" where "vgic_lr_C_Ptr \<equiv> Ptr"
```

Comment:
yay value_type! so glad we made it, and a fun memory

Reply:
and it keeps giving :-)